README.md

```diff
@@ -0,0 +1,153 @@
+python-inspector - inspect Python packages dependencies and metadata
+=====================================================================
+
+Copyright (c) nexB Inc. and others.
+SPDX-License-Identifier: Apache-2.0
+Homepage: https://github.com/aboutcode-org/python-inspector and https://www.aboutcode.org/
+
+``python-inspector`` is a collection of utilities to:
+
+- resolve PyPI packages dependencies
+
+- parse various requirements.txt files and setup.py files as input
+  for resolving dependencies.
+
+- parse additionally various manifests and packages files such as
+  Pipfile, pyproject.toml, poetry.lock and setup.cfg and legacy and
+  current metadata file formats for eggs, wheels and sdist. These
+  have not been wired with the command line yet.
+
+- query PyPI JSON and simple APIs for package information
+
+It grew out of ScanCode toolkit to find and analyze PyPI archives and
+installed Python packages and their files.
+
+The goal of python-inspector is to be a comprehensive library
+that can handle every style of Python package layouts, manifests and lockfiles.
+
+Developing
+----------
+
+- [Install Astral Uv](https://docs.astral.sh/uv/getting-started/installation/). For convenience:
+
+  - Isolated: `pipx install uv`
```

Comment:
I am not against switching to uv as a principle, but that should be its own isolated PR as there will be lots of details to discuss. (And also, in particular and even though this does not matter for this PR, I would not want to assume pipx or pip is installed first). And switching to uv should designed as mostly transparent for users and reusing the existing configure scripts.